config/fcos/v1_0/translate.go

```diff
@@ -17,11 +17,41 @@ package v1_0
 import (
 	"github.com/coreos/butane/config/common"
 	cutil "github.com/coreos/butane/config/util"
+	"github.com/coreos/butane/translate"
 
+	"github.com/coreos/ignition/v2/config/util"
 	"github.com/coreos/ignition/v2/config/v3_0/types"
+	"github.com/coreos/vcontext/path"
 	"github.com/coreos/vcontext/report"
 )
 
+// ToIgn3_0Unvalidated translates the config to an Ignition config. It also
+// returns the set of translations it did so paths in the resultant config
+// can be tracked back to their source in the source config.  No config
+// validation is performed on input or output.
+func (c Config) ToIgn3_0Unvalidated(options common.TranslateOptions) (types.Config, translate.TranslationSet, report.Report) {
+	ret, ts, r := c.Config.ToIgn3_0Unvalidated(options)
+	if r.IsFatal() {
+		return types.Config{}, translate.TranslationSet{}, r
+	}
+
+	for i, disk := range ret.Storage.Disks {
+		// In this case, nothing specifies partition numbers, so match
```

Comment:
Since the first half of this comment has been dropped, "in this case" is now confusing.  I think the main thing here is to stay consistent with later specs, so maybe just `Don't warn if wipeTable is set, matching later spec versions`?